recipes/ggd/meta.yaml

```diff
@@ -0,0 +1,71 @@
+{% set version = "0.1.0" %}
+
+package:
+  name: ggd
+  version: "{{ version }}"
+
+source:
+  url: https://github.com/gogetdata/ggd-cli/archive/v{{ version }}.tar.gz
+  sha256: 540c67b4beb9dee34fa07c55e86e6f58c17dbc345a4700207cbc87615040a7ff
+
+build:
+    number: 0
+    binary_relocation: False
+    noarch: python
+
+requirements:
+  build:
+    - python
+    - requests>=2.20.0
+    - pyyaml
+    - gitpython
+    - git
+    - conda=4.7.12
+    - conda-build=3.18.11
```

Comment:
Does this package really require conda and conda-build internally?

Reply:
Apparently, yes.
It even deliberately uses non-public functions from `conda` which is, let's say "adventurous" xD. But they pin down the version requirements exactly, so if they know what they are downing, that *could be* fine.

Reply:
Yes, this package does use conda and conda-build and non-public functions of conda. Conda acts as the primary infrastructure for our package.